lib/openstack/openstack_event_monitor.rb

```diff
@@ -10,10 +10,13 @@ def self.new(options={})
   end
 
   def self.available?(options)
-    !select_event_monitor_class(options).kind_of? OpenstackNullEventMonitor
+    select_event_monitor_class(options) != OpenstackNullEventMonitor
   end
 
-  DEFAULT_PLUGIN_PRIORITY = 0
+  PLUGIN_PRIORITY_HIGH    = 10
+  PLUGIN_PRIORITY_MED     = 5
+  PLUGIN_PRIORITY_LOW     = 1
+  DEFAULT_PLUGIN_PRIORITY = PLUGIN_PRIORITY_LOW
```

Comment:
Haha!  So, that was the reason it was failing before.  It was sorting with the low numbers first.  And, I changed it here b/c the plugins were using the high numbers as high priority.

I'm fine changing it.  I think having it const'd makes it easier to do that now.